scripts/startup-scripts.sh

```diff
@@ -52,3 +52,15 @@ mpc rescan
 if [ "${READWLANIPYN}" == "ON" ]; then
     /home/pi/RPi-Jukebox-RFID/scripts/playout_controls.sh -c=readwifiipoverspeaker
 fi
+
+#######################
+# Default audio sink to speakers (instead of bluetooth device) irrespective of setting at shutdown
```

Comment:
@ChisSoc 
reading "Default audio sink", I thought you meant "sync". But then I found that there are the terms "sinking" and "sourcing" and - not that I understood it all - that might be the reference. Is it?